views/BD103.html

```diff
@@ -4,13 +4,39 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<meta http-equiv="refresh" content="3; URL=https://bd103.github.io">
-	<title>BD103 Woo</title>
+	<title>BD103</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Pangolin&display=swap');
+    body {
+      font-family: "Montserrat", sans-serif;
+      font-size: 20px;
+      background: linear-gradient(to bottom right, darkcyan, lightcyan);
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-attachment: fixed;
+    }
+    h1, h2 {font-family: "Pangolin", cursive;}
+    .title {font-size: 60px;}
+    .center {text-align: center;}
+    .left {text-align: left;}
+    .right {text-align: right;}
+  </style>
 </head>
 
 <body>
-	<h1>Hi!</h1>
-	<p>I don't have much here, so get redirected in three seconds lol.</p>
+	<h1 class="center title">BD103</h1>
+	<p class="right">Welcome to my page! I am BD103. I am a Python engineer and full-stack website developer (along with other things).</p>
+
+  <h2 class="left">Find me on...</h2>
+  <ol class="right">
+    <li><a href="//repl.it/@BD103">Repl.it</a></li>
+    <li><a href="//github.com/BD103">Github</a></li>
+    <li><a href="//pypi.org/user/BD103">PyPI</a></li>
+    <li><a href="//bd103.github.io">Website</a></li>
+  </ol>
+
+  <h2 class="left">About</h2>
+  <p class="right">I started my programming career in 2017.</p>
```

Comment:
Thanks for ending the redirect, it was horrible. The content of the page is fine for now I think, but maybe add more to the p element like what languages you know, and stuff like that. Also you can just link accounts from stuff like dev.to too. Also, if you are not redirecting, maybe making it open in a new page **_should_** be removed if not done already.